src/sagas/wallet.js

```diff
@@ -147,6 +146,13 @@ export function* startWallet(action) {
   // then we don't know if we've cleaned up the wallet data in the storage
   yield storage.cleanStorage(true, true);
 
+  // Set the server and network as saved on localStorage
+  // If the localStorage is empty, fetch data directly from the lib config
+  const networkName = LOCAL_STORE.getNetwork() || config.getNetwork().name;
+  const serverUrl = LOCAL_STORE.getServer() || config.getServerUrl();
+  config.setServerUrl(serverUrl);
```

Comment:
Does it make sense to set network in the lib config?

Reply:
Good point! Setting the retrieved network name on both the lib and local storage to keep them synchronized. ( https://github.com/HathorNetwork/hathor-wallet/pull/547/commits/03e741100bed24f05664708f85c4b7074b7216dc )